File: src/main/webapp/js/service/blogTypeService.js
app.service("blogTypeService",function($http){
	this.findTypeList = function(){
		return $http.get("./blogType/getTechnologyTypeList.do");
	}
	
	this.findBlogTypeByTypeId = function(typeId){
		return $http.get("./blogType/getBlogTypeByTypeId.do?typeId="+typeId);
	}
	
});

File: src/main/java/com/blog/service/BlogTypeService.java
package com.blog.service;

import java.util.List;

import com.blog.pojo.BlogType;

public interface BlogTypeService {
	/**
	 * 查询分类列表
	 * @return
	 */
	public List<BlogType> getTypeList();

	/**
	 * 根据分类id查询
	 * @param typeId
	 * @return
	 */
	public BlogType getBlogTypeByTypeId(int typeId);

}
